cdap-app-templates/cdap-etl/cdap-etl-api/src/main/java/io/cdap/cdap/etl/api/relational/StringExpression.java

```diff
@@ -0,0 +1,90 @@
+/*
+ * Copyright © 2023 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package io.cdap.cdap.etl.api.relational;
+
+import java.util.Objects;
+import javax.annotation.Nullable;
+
+/**
+ * A default implementation of Expression that simply stores the expression as a string.
+ * The string is assumed to be ANSI SQL compliant.
+ * It is the responsibility of the factory creating {@link StringExpression} objects to enforce correctness of
+ * the expression.
+ */
+public class StringExpression implements ExtractableExpression<String> {
```

Comment:
Please make it final. I don't want to handle inheritance mutations in the api

Reply:
Done.